test/cljx/schema/core_test.cljx

```diff
@@ -277,6 +277,23 @@
       (is (= '(conditional odd? Int weird?)
              (s/explain (s/conditional odd? s/Int 'weird?)))))))
 
+(deftest cond-pre-test
+  (let [s (s/cond-pre
+           s/Int
+           (s/maybe s/Str)
+           (s/cond-pre s/Keyword {:x s/Int})
+           (s/both [s/Num] (s/pred (fn [xs] (even? (count xs))) 'even-len?)))]
```

Comment:
is this predicate really relevant to `cond-pre` in the test?  It might be moreso if there was another sequence schema afterwards.